### DISCUSSION TITLE
灰犀牛【房地产】后面跟着一大群黑天鹅

### DISCUSSION THREAD

**Discussion by frshgrass** [WeNeedHome/SummaryOfLoanSuspension#1033] [General]
其实怎么说呢 已经在历史的关口了 有的人还不自知 腆着脸赖着

韩国和台湾都是在经历了经济高速发展 贫富差距加大后 开始由原来的威权体制转向民主 韩国有光州事件 台湾有美丽岛事件 

大陆也在这个进程中 而且基本可以确定就是房地产垮掉后 全民财富蒸发 大量失业后 人民开始转向上街 

经济规律是不会消失的 可以人为推迟 但是会以更加迅猛败坏的方式到来（美国上世纪禁酒令，结果反而更多人酗酒，而且真正的做大美国黑帮）

所以泡沫一定是会破的 只不过财富蒸发后 大面积失业后 这股社会不满的情绪要怎么发泄：

  1，结束集权，建立民主（同为儒家文化圈，同为威权体制时期经历经济高速发展，极大可能和韩台一样，经历民主运动后开始民主选举）

  2，借由这些年培养的小粉红学生，文革回潮。但是这是相当恐怖的，没有人有足够的权威一定能独善其身，因为能广泛打出来的就是毛的旗号，借毛是权威，各派血斗。有人说自己是领袖那只是他自以为是，他没有这样的权威（文革时期只有毛作为开国皇帝是安全的，现在文革谁也说不好谁安全。各种泡打司令部，各种红旗打红旗，就像伊斯兰各种教派血斗）但是实话说现在的小粉红品客 还不够多 这种概率比较小 

也终于快到了 解下束缚在大家身上枷锁的时刻，之后 大家可以松一口气 繁衍生息 。

**Comment by TomMao-CN** [comment]
想什么的？看看鹅城的百姓

**Reply by frshgrass** [reply]
发展规律而已 要不然大清为什么无了 逆历史潮流的 螳臂当车久不了

**Reply by TomMao-CN** [reply]
> 发展规律而已 要不然大清为什么无了 逆历史潮流的 螳臂当车久不了

武器在老爷们手里，哎

**Reply by xujiyou** [reply]
> 

兵依旧是子弟兵

**Comment by JayChou55** [comment]
想太多了。。。现在的舆论管控程度 超出你想象 。。我就是干舆情监管的开发的

**Reply by frshgrass** [reply]
美丽新世界？那也得有钱来搞呀 历史上很多朝代都亡于财政 就现在经济和房地产的程度 以及未来老年化的趋势 大清地主家也没有余粮呀

**Reply by Aizenz** [reply]
无论是网络舆情管控还是线下物理维权都需要足够的财政实力予以支持，在地产包税制崩溃的浪潮之下，这些维系的力量也会随之衰减。三个月不发工资，最忠诚的忠诚都会怀疑人生的。

**Reply by tdmm** [reply]
能具体说说么

**Comment by sthfake** [comment]
同意正文，主题灰犀牛后面一群灰犀牛是否更妥？

**Comment by Siryedu** [comment]
断贷这个事情本来就不能看成是一个独立的经济事件，干了30年房地产，政府、开发商、业主欠一屁股债，钱去哪里了？

**Comment by kajsnlk** [comment]
别太乐观，韩国、台湾能成功转向民主化，主要还是军事受美国等西方制约，凡事有军权在手的专权哪个是会妥协于民众的，坦克一出来，按照社会大多数的麻木心态，大不了就是30年前在来一遍，走向朝鲜化俄罗斯化。除非腐败到军队都管不住了，军政府才会垮台。

**Comment by toxin08x** [comment]
楼主太乐观了。有什么样的政府就有什么样的人民。就这国内培养的战狼思维，就不配得到民主体制。能润赶紧润吧。

**Reply by Flywolfs** [reply]
sb吧，哪来的狗崽子，git都没怎么用过跑这儿来叫，你这德性的配？

**Reply by toxin08x** [reply]
> sb吧，哪来的狗崽子，git都没怎么用过跑这儿来叫，你这德性的配？

诶呦你用过几次git就给这优越了？我润到美国享受民主体制快9年了。国内全国性的污名化民主（皿煮）这种行为，习近平连任，选票走过场这就是对民主最大的亵渎。配不配不是取决于说了什么，而是国内都做了什么。

**Reply by Flywolfs** [reply]
哎呦看把你能的，你是牛逼！

**Reply by tanuk1997** [reply]
> 哎呦看把你能的，你是牛逼！

他确实比你牛逼，你只能无能狂怒，被晶哥城管暴打一顿你也只能跪下来求青天大老爷

**Comment by TALK1949** [comment]
89年和平转型的窗口期也是以血腥镇压而告终，我对在军国主义教育下成长起来的新一代不抱期望。

**Reply by oceanzhang860411** [reply]
從現在拼凑出來的歷史脈絡，89年基本也是不可能成功的，因爲當時的學生根本不知道自己要什麽，包括當時廣場上學生在最後的時刻還有意見分歧。

**Reply by slowmna** [reply]
学生虽然有些激进有些分歧, 但是胡耀邦赵紫阳在斗争中被击败才是主因吧. 说起来真的是错失了建国后最近的一次民主改革机会, 当前提的政治改革, 现在成了纯粹的口号

**Reply by oceanzhang860411** [reply]
胡耀邦和趙紫陽的鬥爭失敗是一部分原因，但那時候學生是掌握主動權的，連日人民報都是站在學生這邊的，整體的社會輿論包擴國際形式也是有利的，如果最後時刻學生按照最後一次會議達成的一致，暫時從廣場撤退，民主化的思想可能就能播下種子。千百個無形的敵人，不比廣場上1萬個抗議學生殺傷力更大嗎？

**Comment by fnnnnng** [comment]
我也觉得乐观了，封建冷兵器时期人数起来了还能撼动一下，如今你人再多我压死两个，其他人就退了

**Comment by Pacio** [comment]
1完全不可能~
最大的可能就是2，文革2.0
具体闹多大，我觉得冯小刚在三人行里面说的挺合适的

**Comment by oceanzhang860411** [comment]
經濟泡沫破裂肯定是不能逆轉，但是上街就能民主也是幻想了。就在這個討論區，就能看到那麽多懷念毛臘肉，覺得國家是好的，受罪后依然愛國的，就應該明白，這群烏合之衆是不可能真正明白，先要保障私有財產，保障法律才會有美好生活。
不要説什麽文革2.0，如果真的回到文革，這次恐怕慘烈程度前所未有。

**Reply by gitaa1** [reply]
So true!

**Comment by shawn-yee** [comment]
二舅偶像是毛主席
觉醒?
一百年以后。
